pocs/state/states/default/pointing.py

```diff
@@ -94,7 +94,11 @@ def on_enter(event_data):
                         if observation.field is not None:
                             pocs.logger.debug("Slewing back to target")
                             pocs.observatory.mount.set_target_coordinates(observation.field)
-                            pocs.observatory.mount.slew_to_target()
+                            pocs.observatory.mount.slew_to_target(blocking=True)
+
+                    if img_num == (num_pointing_images - 1):
+                        pocs.logger.info(f'Separation outside threshold but at max corrections. ' +
+                                         'Will proceed to observations.')
```

Comment:
If separation is largely than some threshold after `num_pointing_images` then the system should **not** proceed but should shut down.